src/main/webapp/app/assessment-shared/assessment-layout/assessment-layout.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="assessment-container">
+    <jhi-assessment-header
+        [showBackButton]="showBackButton"
+        (navigateBack)="navigateBack.emit()"
+        [isLoading]="isLoading"
+        [busy]="busy"
+        [isAssessor]="isAssessor"
+        [isAtLeastInstructor]="isAtLeastInstructor"
+        [canOverride]="canOverride"
+        [assessmentsAreValid]="assessmentsAreValid"
+        [result]="result"
+        (save)="save.emit()"
+        (submit)="submit.emit()"
+        (cancel)="cancel.emit()"
+        (nextSubmission)="nextSubmission.emit()"
+        [hasConflict]="conflicts != null"
```

Comment:
Please try to avoid `!=` and use `!==` to avoid casting and strictly check equality